src/contradictory_claims/cli.py

```diff
@@ -124,16 +131,18 @@ def main(train, report):
 
     if report:
         eval_data_dir = os.path.join(root_dir, "input")
-        eval_data_path = os.path.join(eval_data_dir, "drug_individual_claims_similarity_annotated.xlsx")
-        active_sheet = "drug_individual_claims_similari"
+        # eval_data_path = os.path.join(eval_data_dir, "drug_individual_claims_similarity_annotated.xlsx")
+        # active_sheet = "drug_individual_claims_similari"
+        eval_data_path = os.path.join(eval_data_dir, "Pilot_Contra_Claims_Annotations_06.30.xlsx")
```

Comment:
Hmm..should this be renamed without the date so that we avoid having to update the file in the future?

Reply:
Because this is specifically the annotations we received from 6/30 I'm okay with this staying the same. There should eventually maybe be an argument so these paths aren't hard coded in there, but I think we can deal with that as we polish more